src/Symfony/Component/DependencyInjection/Tests/Dumper/YamlDumperTest.php

```diff
@@ -62,8 +64,10 @@ public function testDumpAutowireData()
         $this->assertStringEqualsFile(self::$fixturesPath.'/yaml/services24.yml', $dumper->dump());
     }
 
-    private function assertEqualYamlStructure($yaml, $expected, $message = '')
+    private function assertEqualYamlStructure($expected, $yaml, $message = '')
```

Comment:
It works either way but imo this function should always be executed with the same flags used in `YamlFileLoader`.